packages/react/src/index.ts

```diff
@@ -91,15 +91,22 @@ Object.defineProperties(Signal.prototype, {
 });
 
 // Track the current owner (roughly equiv to current vnode)
-// let currentOwner: ReactOwner;
-// Object.defineProperty(internals.ReactCurrentOwner, "current", {
-// 	get() { return currentOwner; },
-// 	set(owner) { currentOwner = owner; },
-// });
+let lastOwner: ReactOwner;
+let currentOwner: ReactOwner;
+Object.defineProperty(internals.ReactCurrentOwner, "current", {
+	get() {
+		return currentOwner;
+	},
+	set(owner) {
+		currentOwner = owner;
+		if (currentOwner) lastOwner = currentOwner;
+	},
+});
```

Comment:
`currentOwner` would be better defined as `ReactOwner | null` as per the `React.RefObject` interface.